src/main/java/core/basesyntax/impl/ConvertToCsvServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ConvertToCsvService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConvertToCsvServiceImpl implements ConvertToCsvService {
+    private static final String[] HEADER = {"fruits", "quantity" + System.lineSeparator(),};
+    private static final int STORAGE_SIZE = Storage.fruits.size();
```

Comment:
you don't need this variable. (Firstly, you can use simple `Storage.fruits.size()`, if you need to get a size. Secondly, you can use Map.Entry to go through map pairs.